packages/theming/android-theme/src/colorsTokens.ts

```diff
@@ -1,11 +1,13 @@
 import { ThemeColorDefinition } from '@fluentui-react-native/theme-types';
 import { FluentUIAndroidPalette } from './colorsSemantic';
+import { createColorAliasTokens } from './createAliasTokens';
 
-/** creates a palette of colors for the android theme, given the FluentUI Android Palette */
+/** creates a palette of colors for the android theme, given the FluentUI Android Palette - ??.
+ * What is the rest if the palette color?
```

Comment:
Intentional?

Reply:
Kind of - it was something I wanted to ask on the PR - but not as a check-in comment inline.